README.yaml

```diff
@@ -158,39 +158,39 @@ examples: |-
   We use YAML for the configuration files in order to separate configuration settings from business logic. It's also a portable format that can be used across multiple tools. Our convention is to name files by `$env-$stage.yaml` (e.g. `ue2-testing.yaml`), so for example an `$env` could be `ue2` (for `us-east-2`) and the `$stage` might be `testing`. Workspace names are derived from the `$env-$stage-$component`, which looks like  `ue2-testing-eks`.
 
   ```yaml
+  # These are variables assigned to all component stacks
+  vars:
+    my_global_var: value1
+
   # Components are all the top-level root modules
   components:
-    # Globals are exported as TF_VAR_... environment variables in every workspace
-    globals:
-      # Used to determine the name of the workspace (e.g. the 'testing' in 'ue2-testing')
-      stage: testing
-      # Used to determine the name of the workspace (e.g. the 'ue2' in 'ue2-testing')
-      environment: ue2
-    # The configuration file format is designed to be used across multiple tools.
-    # All terraform components should be listed under this section.
     terraform:
       # List one or more Terraform components here
       first-component:
-        # Controls whether or not this workspace should be created
-        # NOTE: If set to 'false', you cannot reference this workspace via `triggers` in another workspace!
-        workspace_enabled: true
-        # Override the version of Terraform for this workspace (defaults to the latest in Spacelift)
-        terraform_version: 0.13.4
-        # Controls the `auto_apply` setting within this workspace
-        auto_apply: true
-        # Add extra 'Run Triggers' to this workspace, beyond the parent workspace, which is created by default
-        # These triggers mean this component workspace will be automatically planned if any of these workspaces are applied.
-        triggers:
-          - uw2-testing-example2
-          - gbl-root-example1
+        settings:
+          spacelift:
+            # Controls whether or not this workspace should be created
+            # NOTE: If set to 'false', you cannot reference this workspace via `triggers` in another workspace!
+            workspace_enabled: true
+            # Override the version of Terraform for this workspace (defaults to the latest in Spacelift)
+            terraform_version: 0.13.4
+            # Controls the `auto_apply` setting within this workspace
+            auto_apply: true
+            # Add extra 'Run Triggers' to this workspace, beyond the parent workspace, which is created by default
+            # These triggers mean this component workspace will be automatically planned if any of these workspaces are applied.
+            triggers:
```

Comment:
I think this is out of date as well, but I don't recall the new format. Does someone have it handy so we can update this example?